tests/js/spec/views/onboarding/index.spec.jsx

```diff
@@ -35,38 +33,5 @@ describe('OnboardingWizard', function() {
       });
       expect(wrapper).toMatchSnapshot();
     });
-
-    it('should render and respond to click events', function() {
```

Comment:
What's the reason for removing this?

Reply:
This test was more of an integration test, really just verifying that it renders it's children. The `CreateProject` component got a bit more complicated and needed more than this provided to work with the HoCs it uses, and it didn't seem like this test was providing much value. So I removed it.

Reply:
The test probably has limited value, but seeing as how it's a pretty simple integration test, our testing infrastructure ideally would be able to support it. Can we invest a bit to making this test work with the HoCs? `withApi` and `withOrganization` should be stubbed and supported already

Reply:
Alright. Sounds good.

Reply:
I'm going to clean this up in a second PR actually. The naming of this stuff is just all over the place, so I'm going to try and make the component structure more consistent, in which case I think the test we add back in here can be simple.